build_tools/bump_submodules.py

```diff
@@ -170,7 +235,32 @@ def run(args: argparse.Namespace, fetch_args: list[str], system_projects: list[s
             ["git", "push", "-u", "origin", args.branch_name],
             cwd=THEROCK_DIR,
         )
-
+        new_hashes = get_hashes_for_component(component)
+        old_h, new_h, changed_path = detect_hash_change(old_hashes, new_hashes)
+        if not old_h or not new_h:
+            log("WARNING: No submodule hash changes detected for the component.")
+            pr_title = f"Bump {component} from unknown to unknown"
```

Comment:
If there are no new changes to https://github.com/ROCm/rocm-libraries, this script should not create a commit, push a branch, or create a PR.

I don't think we should have any "unknown to unknown" PRs.